includes/Modules/Analytics/Advanced_Tracking/Event_List_Registry.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Analytics\Advanced_Tracking\Event_List_Registry
+ *
+ * @package   Google\Site_Kit\Modules\Analytics
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Analytics\Advanced_Tracking;
+
+use Google\Site_Kit\Modules\Analytics\Advanced_Tracking\Measurement_Events\Measurement_Event_List;
+
+/**
+ * Class for registering third party event lists.
+ *
+ * @since n.e.x.t.
+ * @access private
+ * @ignore
+ */
+class Event_List_Registry {
+
+	/**
+	 * The list of active event lists.
+	 *
+	 * @since n.e.x.t.
+	 * @var Measurement_Event_List[]
+	 */
+	private $active_event_lists;
+
+	/**
+	 * Event_List_Registry constructor.
+	 *
+	 * @since n.e.x.t.
+	 */
+	public function __construct() {
+		$this->active_event_lists = array();
+	}
+
+	/**
+	 * Registers a third party event lists.
+	 *
+	 * @since n.e.x.t.
+	 *
+	 * @param Measurement_Event_List $event_list The third party event list to be registered.
+	 * @throws \Exception Thrown when $event_list is not an instance of Measurement_Event_List.
+	 */
+	public function register( Measurement_Event_List $event_list ) {
+		if ( ! $event_list instanceof Measurement_Event_List ) {
+			throw new \Exception( 'Event list must extend Measurement_Event_List.' );
+		}
+
+		$event_list->register();
+		$this->active_event_lists[] = $event_list;
+	}
+
+	/**
+	 * Gets the list of active event lists.
+	 *
+	 * @since n.e.x.t.
+	 *
+	 * @return Measurement_Event_List[] The list of active event lists.
+	 */
+	public function get_active_event_lists() {
```

Comment:
Maybe shorten to more generic `get_all()`? Since this is only a registry for event lists, no need to duplicate that in the method name. We can keep it short like in `register()`.

Reply:
Resolved.